docker/Dockerfile-circleci

```diff
@@ -14,4 +14,4 @@ RUN wget -q https://bootstrap.pypa.io/get-pip.py && \
 RUN pip install tensorboard opencv-python-headless
 ARG PYTORCH_VERSION
 ARG TORCHVISION_VERSION
-RUN pip install torch==${PYTORCH_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/cu101/torch_stable.html
+RUN pip install torch==${PYTORCH_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/cu101/torch_stable.html -f https://download.pytorch.org/whl/test/cu101/torch_test.html
```

Comment:
after stable is released, will this command install 1.6 stable or still 1.6 testing?

Reply:
I believe this should prioritize installs in order of when the indices are noted, for example it should pick out the 1.6 version (if available) in `torch_stable.html` before it picks out the version in `torch_test.html`